src/api/index.js

```diff
@@ -0,0 +1,46 @@
+import { apiBaseUrl, requestDefaultHeaders } from "../../config";
+
+export const getTodoList = async () => {
+  const res = await fetch(`${apiBaseUrl}/todos`);
+  const data = await res.json();
+  return data.data;
+};
+
+export const addTodo = async ({ text }) => {
+  const res = await fetch(`${apiBaseUrl}/todos`, {
+    method: "POST",
+    headers: requestDefaultHeaders,
+    body: JSON.stringify({ text }),
+  });
+  const data = await res.json();
+  return data.data;
+};
+
+export const deleteTodo = async (taskId) => {
```

Comment:
Either go with separate parameters or an object as a param. But what is important is that you follow the same convention across the whole code base. I personally prefer the object param for flexibility reasons.
```suggestion
export const deleteTodo = async ({ taksId }) => {
```

Reply:
Done.